package.json

```diff
@@ -61,7 +61,7 @@
     "@types/classnames": "^2.3.1",
     "@types/deep-equal": "^1.0.1",
     "@types/express": "^4.17.17",
-    "@types/http-proxy-middleware": "^0.19.3",
+    "@types/http-proxy-middleware": "1.0.0",
```

Comment:
Denne trengs ikke for v2 :) 
https://www.npmjs.com/package/@types/http-proxy-middleware
> This is a stub types definition. http-proxy-middleware provides its own type definitions, so you do not need this installed.